grid/websocket_client.py

```diff
@@ -0,0 +1,87 @@
+import binascii
+import time
+from typing import List
+from typing import Union
+
+import socketio
+import torch
+
+import syft as sy
+from syft.frameworks.torch.tensors.interpreters import AbstractTensor
+from syft.workers import BaseWorker
+
+
+class WebsocketGridClient(BaseWorker):
+    """ Websocket Grid Client """
+
+    def __init__(
+        self,
+        hook,
+        addr: str,
+        id: Union[int, str] = 0,
+        is_client_worker: bool = False,
+        log_msgs: bool = False,
+        verbose: bool = False,
+        data: List[Union[torch.Tensor, AbstractTensor]] = None,
+    ):
+        """
+        Args:
+            hook (sy.TorchHook): a normal TorchHook object
```

Comment:
tip: if you're using typing, I don't think types are needed at docstrings.

Reply:
Fixed at ebc7c81